device-types/Dell/PowerConnect_8024.yaml

```diff
@@ -0,0 +1,70 @@
+manufacturer: Dell
+model: PowerConnect 8024
+slug: powerconnect_8024
+part_number: PC8024
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: Serial
+    type: de-9
+power-ports:
+- name: PSU0
+  type: iec-60320-c14
+  maximum_draw: 300
+  allocated_draw: 89
+- name: PSU1
+  type: iec-60320-c14
+  maximum_draw: 300
+  allocated_draw: 89
+interfaces:
+  - name: Te1/0/1
+    type: 10gbase-x-sfpp
+  - name: Te1/0/2
+    type: 10gbase-x-sfpp
+  - name: Te1/0/3
+    type: 10gbase-x-sfpp
+  - name: Te1/0/4
+    type: 10gbase-x-sfpp
+  - name: Te1/0/5
+    type: 10gbase-x-sfpp
+  - name: Te1/0/6
+    type: 10gbase-x-sfpp
+  - name: Te1/0/7
+    type: 10gbase-x-sfpp
+  - name: Te1/0/8
+    type: 10gbase-x-sfpp
+  - name: Te1/0/9
+    type: 10gbase-x-sfpp
+  - name: Te1/0/10
+    type: 10gbase-x-sfpp
+  - name: Te1/0/11
+    type: 10gbase-x-sfpp
+  - name: Te1/0/12
+    type: 10gbase-x-sfpp
+  - name: Te1/0/13
+    type: 10gbase-x-sfpp
+  - name: Te1/0/14
+    type: 10gbase-x-sfpp
+  - name: Te1/0/15
+    type: 10gbase-x-sfpp
+  - name: Te1/0/16
+    type: 10gbase-x-sfpp
+  - name: Te1/0/17
+    type: 10gbase-x-sfpp
+  - name: Te1/0/18
+    type: 10gbase-x-sfpp
+  - name: Te1/0/19
+    type: 10gbase-x-sfpp
+  - name: Te1/0/20
+    type: 10gbase-x-sfpp
+  - name: Te1/0/21
+    type: 10gbase-tx
+  - name: Te1/0/22
+    type: 10gbase-tx
+  - name: Te1/0/23
+    type: 10gbase-tx
+  - name: Te1/0/24
+    type: 10gbase-tx
+  - name: Management
+    type: 100base-tx
+    mgmt_only: true
```

Comment:
The validation test is failing because we do not have a "10gbase-tx" interface type defined.  I think you meant this to be "10gbase-t".  Fix the issue and I will merge this PR.

Reply:
There are a couple of other possible issues with this template.  I have a PowerConnect 8024 so I thought I would try this.

I believe the SFPP and 10gbase-T ports are reversed.  My own switch has 20 10gbase-T ports and 4 sfpplus.
Also, it's nitpicky but the Management interface is 1000base-tx.

It's possible there are different PC8024 models.   This is what it shows in show version for mine:

```
System Object ID.................. 1.3.6.1.4.1.674.10895.3023
System Model ID................... PC8024
Machine Type...................... Powerconnect 8024
```

Reply:
@rfdrake do you want to take this PR over?